ikarus/finiteelements/mechanics/enhancedassumedstrains.hh

```diff
@@ -211,16 +190,16 @@ forward the
     if constexpr (Traits::mydim == 2) {
       switch (numberOfEASParameters) {
         case 0:
-          easVariant_ = std::monostate();
+          easVariant_.var = EAS::Q1E0(localView().element().geometry());
```

Comment:
This is not so nice to access this datamember. This should be private: https://godbolt.org/z/q4feKa368 I imagine to have the variant wrapper like this. Then it is directly constructable from its underlying types.